lib/ar2dto/converter.rb

```diff
@@ -18,7 +18,7 @@ def serializable_hash
     private
 
     def apply_global_configs(options)
-      options[:except] = AR2DTO::Config.instance.except + Array(options[:except])
+      options[:except] = Array(model.class.ar2dto.except) + Array(options[:except])
```

Comment:
```suggestion
      options[:except] = Array(model.class.ar2dto.except) | Array(options[:except])
```

wdyt?

Reply:
I love it!

Reply:
now that I'm thinking.. how should this behave? e.g. we have global excepts `updated_at`, then model excepts `created_at` and then `.to_dto(except: :name)` should we exclude the three? or should we only exclude name?